tests/dialects/test_hive.py

```diff
@@ -176,6 +176,44 @@ def test_ddl(self):
             },
         )
 
+        self.validate_identity("ALTER TABLE x PARTITION(y = z) ADD COLUMN a VARCHAR(10)")
+        self.validate_identity(
+            "ALTER TABLE x CHANGE a a VARCHAR(10)",
+            write_sql="ALTER TABLE x CHANGE COLUMN a a VARCHAR(10)",
```

Comment:
We don't generally use the `write_sql` keyword argument.
```suggestion
            "ALTER TABLE x CHANGE COLUMN a a VARCHAR(10)",
```